Sources/Datadog/RUM/Integrations/CrashReportReceiver.swift

```diff
@@ -326,7 +331,7 @@ internal struct CrashReportReceiver: FeatureMessageReceiver {
             application: .init(id: lastRUMView.application.id),
             ciTest: lastRUMView.ciTest,
             connectivity: lastRUMView.connectivity,
-            context: nil,
+            context: errorContext,
```

Comment:
/suggestion I think we can pass the `lastRUMView.context` directly here, no?
```suggestion
            context: lastRUMView.context,
```

Reply:
Indeed, no need for the extra copy.